GetStream/stream-chat-android
# [2880] Remove ExoPlayer dependency from Compose

### 🎯 Goal

The Compose part of getting rid of ExoPlayer dependency. 

### 🛠 Implementation details

Using out of the box `ViewView`, which in turn uses `MediaPlayer` and `SurfaceView` under the hood. Besides that, `VideoView` provides basic controls. 

### 🎨 UI Changes

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/9600921/154507460-5b87b65f-66c4-48ea-a298-25235a0b1790.mp4" controls="controls" muted="muted" />
</td>
<td>
<video src="https://user-images.githubusercontent.com/9600921/154648069-25a86dae-ce24-46a7-9624-b3015308da05.mp4" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>

### 🧪 Testing

1. Run Compose sample app
2. Navigate to the message list
3. Tap on a video / audio attachment to see media preview

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_
